avocado/utils/network/interfaces.py

```diff
@@ -21,6 +21,7 @@
 import os
 import shutil
 from ipaddress import ip_interface
+from avocado.utils import process
 
 from avocado.utils.distro import detect as distro_detect
```

Comment:
please consider moving 

`from avocado.utils import process` after this import statement

Reply:
Done